CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/visual/InteractiveVKVisualProcessor.java

```diff
@@ -162,6 +162,7 @@ public void apply() {
 
     @Override
     public VisualOperation setHitTestingEnabled(boolean enabled) {
+        //throw new UnsupportedOperationException("setHitTestingEnabled shouldn't be needed for Vulkan");
```

Comment:
can remove this commented out code

Reply:
Actually I left that in there to remind one of us to go and rationalise the way we handle hit testing.  Can pull it now if you want, but it could be good to do in your hit test PR?

Reply:
No problem